eth/stubclient.go

```diff
@@ -98,8 +102,18 @@ func (e *StubClient) GetDelegatorUnbondingLock(addr common.Address, unbondingLoc
 func (e *StubClient) GetTranscoderEarningsPoolForRound(addr common.Address, round *big.Int) (*lpTypes.TokenPools, error) {
 	return nil, nil
 }
-func (e *StubClient) RegisteredTranscoders() ([]*lpTypes.Transcoder, error) { return nil, nil }
-func (e *StubClient) IsActiveTranscoder() (bool, error)                     { return false, nil }
+func (e *StubClient) RegisteredTranscoders() ([]*lpTypes.Transcoder, error) {
+	var addresses = []string{"https://127.0.0.1:8936", "https://127.0.0.1:8937", "https://127.0.0.1:8938"}
+	var transcoders []*lpTypes.Transcoder
+
+	for _, addr := range addresses {
+		transc := &lpTypes.Transcoder{ServiceURI: addr}
+		transcoders = append(transcoders, transc)
+	}
+
+	return transcoders, nil
```

Comment:
This is OK for now. The general pattern we follow for stubbing is:

Define `StubClient` with a list of `Transcoders` (technically `Orchestrators`), eg

```type StubClient struct {
   ...
   Orchestrators []*lpTypes.Transcoder,
   ...
}
```

As necessary, initialize the `Orchestrators` list as necessary during testing time. Eg, 
```
func TestOnchainOrchestratorPool() {
  var orchestrators []*lpTypes.Transcoder
  // populate `orchestrators` list
  stubclient := eth.StubClient{Orchestrators: orchestrators}
  n := NewLivepeerNode(stubclient, "", nil)
... 
}
```

The rest of the time, the Orchestrators list can be left unset as it doesn't affect the remainder of the test cases using `StubClient`. This way we can adjust the list of orchestrators as needed to match the expectations of the test (eg, perhaps checking the case where we have zero onchain orchestrators.)

Reply:
That makes a lot of sense. I didn't feel very comfortable adding a hard-coded value to `RegisteredTranscoders` in `StubClient`. This is a better way of doing it. I've changed it!